tests/transactional/install_updates.pm

```diff
@@ -39,7 +39,8 @@ sub run {
     # Now we add the incident repositories and do a zypper patch
     add_test_repositories;
     record_info('Updates', script_output('zypper lu'));
-    my $ret = trup_call('up', timeout => 300, proceed_on_failure => 1);
+    my $ret = 0;
+    $ret = trup_call('up', timeout => 300, proceed_on_failure => 1) unless get_var('DISABLE_UPDATE_WITH_PATCH');
```

Comment:
`my $ret = ...` in the same line.

Reply:
In same line, CI will report issue that define shouldn't be in condition, so I define it in advance.

Reply:
Yes, just found it never used. Updated.